com.openwall.John.appdata.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) 2019 Claudio André <claudioandre.br at gmail.com> -->
+<component type="console-application">
+  <id>com.openwall.John</id>
+  <metadata_license>GFDL-1.3</metadata_license>
+  <project_license>GPL-2.0</project_license>
+
+  <name>John the Ripper Community Edition</name>
+  <summary>John the Ripper "Jumbo" password cracker</summary>
+  <description>
+    <p>
+       John the Ripper is a fast password cracker. Its primary purpose is to
+       detect weak Unix passwords. Besides several crypt(3) password hash types,
+       supported out of the box are Windows LM hashes, plus lots of other hashes
+       and ciphers types.
+
+       Read more at:
+       - github.com/magnumripper/JohnTheRipper
+       - rebrand.ly/JtRPackage
+
+       This version integrates lots of contributed patches including GPU support.
+    </p>
+  </description>
+
+  <icon type="local">/app/share/icons/com.openwall.John.png</icon>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/claudioandre-br/packages/raw/master/john-the-ripper/metadata/help.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/claudioandre-br/packages/raw/master/john-the-ripper/metadata/build-info.png</image>
+    </screenshot>
+  </screenshots>
+
+  <url type="homepage">https://www.openwall.com/john/</url>
+  <url type="bugtracker">https://github.com/magnumripper/JohnTheRipper/issues</url>
+  <url type="donation">https://www.zerodayclothing.com/openwall_store.php</url>
+
+  <developer_name>Openwall and others</developer_name>
+  <update_contact>claudioandre.br@gmail.com</update_contact>
+
+  <categories>
```

Comment:
Categories should be placed only on the desktop file, they are later merged using appstream-compose.